js/build.js

```diff
@@ -0,0 +1,14 @@
+require('esbuild').build({
+    entryPoints: [
+        'src/viewer.js',
+        // other files you want to end up in /static
+    ],
+    format: 'esm',
+    outdir: '../static/',
+    bundle: true,
+    splitting: true,
+    minify: process.env.NODE_ENV === 'production', // only minify if production build
```

Comment:
easier for debugging

Reply:
I removed the `--minify` somewhere at one point when debugging. But this makes sense that you can have both modes.